tests/test_live_cp_download.py

```diff
@@ -210,15 +211,15 @@ def test_gsutil_vs_gswrap_download_non_recursive(self):  # pylint: disable=inval
                 tests.common.call_gsutil_cp(
                     src=test_case[0], dst=test_case[1], recursive=False)
                 gsutil_paths = tests.common.ls_local(path=ls_path)
-                [gsutil_ls_set.add(path) for path in gsutil_paths]
+                gsutil_ls_set.union([path for path in gsutil_paths])
```

Comment:
couldn't you just write `gsutil_ls_set.union(gsutil_paths)`? (Please also fix elsewhere)

Reply:
You're totally right... I can't believe that I didn't see it. Thank you! :)